bigtable/integration_test.go

```diff
@@ -2033,6 +2033,26 @@ func TestIntegration_AdminBackup(t *testing.T) {
 	table := testEnv.Config().Table
 	cluster := testEnv.Config().Cluster
 
```

Comment:
Are there unit tests that cover RestoreTable currently or only integration tests? I don't see any unit tests that call RestoreTable directly but just want to make sure I'm not missing something. @kolea2